client/src/context/UserContext.js

```diff
@@ -1,10 +1,12 @@
 import { createContext } from 'react';
 
-export default createContext({
+export const UserContext = createContext({
```

Comment:
Could we create different `Context`'s for different things (i.e., UserContext, SocketContext)?